.github/README.md

```diff
@@ -1,38 +1,38 @@
 # Q-CTRL Back-end Challenge
 
-The Q-CTRL Back-end Challenge is a way for applicants to roles within the Q-CTRL Back-end Engineering team, to demonstrate their skills and overall approach to back-end application development.
+The Q-CTRL Back-end Challenge is a way for applicants to roles within the Q-CTRL Back-end Engineering team to demonstrate their skills and overall approach to solving a challenge based upon a set of requirements.
 
-The requirements are intentionally sparse to allow some flexibility. We don't want you to do a lot, but what you do should be your best work.
+The requirements are intentionally sparse (the devil is in the details). We don't want you to do a lot, but what you do should be your best work and should clearly demonstrate you embody [the three virtues](https://thethreevirtues.com/).
 
-## Table of Contents
+## The Rules
 
-  - [The Rules](#the-rules)
-  - [The Challenge](#the-challenge)
-  - [Contributing](#contributing)
-  - [Credits](#credits)
-  - [License](#license)
+- Read and understand [the challenge](#the-challenge).
+- Create a solution that satisfies [the requirements](#the-requirements).
+- Send a link to the repository containing your solution to the person who contacted you.
 
-## The Rules
+**IMPORTANT:** The repository containing your solution must be public, must contain a `README.md` file, **MUST NOT** be a fork of this repository, and **MUST NOT** make any reference to "Q-CTRL" in the repository name, description, or code.
 
-- Read and understand [The Challenge](#the-challenge).
-- Create a solution that satisfies the [Requirements](#requirements).
-- Send the link to the repository containing your solution to the person who contacted you.
+**Note:** You should spend no more than three hours on this challenge.
 
 ## The Challenge
 
-### Background
-
 At Q-CTRL, we use a suite of microservices to support our products. Typically, these microservices are written using a Python web framework and expose a [GraphQL](https://graphql.org/) API so clients and other services can communicate with them.
```

Comment:
@luigibertaco this is not a challenge, it's just a statement of fact. See https://github.com/qctrl/front-end-challenge#the-challenge for a good challenge.